contributing/lts-release.md

```diff
@@ -1,6 +1,8 @@
 # Long-Term Stable (lts) Release Channel
 
-The **lts** release channel provides a previous **stable** build for one-month intervals. Approximately monthly, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
+The **lts** release channel provides a previous **stable** build for one-month intervals. On the second Monday of each month, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
+
+In the event that the second Monday of the month falls during a [release freeze](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#release-freezes) (such as on one of the major US holidays), the promotion will be performed the following day (ie. Tuesday) instead.
```

Comment:
> the promotion will be performed the following day (ie. Tuesday) instead.

"The following day" may not apply if the release freeze extends for multiple days instead of just the Monday. How about saying this?

> the promotion will be performed after the end of the release freeze.

Reply:
Done.